server/routes/applications.js

```diff
@@ -10,25 +10,38 @@ const validator = new Validator();
 const routeUtils = require(`${serverRoot}/routes/utils`);
 const routeViews = 'applications';
 
-router.get('(/manage-applications)?', (req, res, next) => {
-  logger.info(`GET request to serve index page: ${req.path}`);
-  const viewData = {
+function createViewData (title, activePage) {
```

Comment:
We generally preserve the route for handlers only -- and no other methods...  should you find the need for a helper method for the route, that should go into `routes/utils/application.js`...

Reply:
Fair point.

Reply:
in fact, seeing as this is across multiple routes, you could drop it under `routes/utils/index.js`

Reply:
changed.